packages/virtualbox-deb/virtualbox-deb.pacscript

```diff
@@ -5,30 +5,33 @@ pkgver="7.0.16"
 arch=('amd64')
 breaks=("${gives}-bin")
 pkgdesc="VirtualBox is a general-purpose full virtualizer for x86 hardware, targeted at server, desktop and embedded use"
-distro_version= "$(lsb_release -is)~$(lsb_release -cs)"
-url="https://download.virtualbox.org/virtualbox/${pkgver}/virtualbox-7.0_${pkgver}-162802~${distro_version}_amd64.deb"
+codename="${DISTRO#*:}"
 case "${distro_version}" in
-"Debian~bookworm")
-	url
-	hash="6aa43d4713d3a351aa2af5bc86fca452897f312ee7c8b9a26cf6e9a6924494de"
-	;;
-"Debian~bullseye")
-	hash="001ea3bbdf06f7931bb64d7e6b556aef7fe2c3721c882054445580aaab5fc70c"
-	;;
-"Debian~buster")
-	hash="5bccc51b1b0f0768872e0037eeae6330fc488a61969203a610c0005aeb2bd53d"
-	;;
-"Ubuntu~bionic")
-	hash="a8cf2d9aecfba9170c67d39dd0368410ff19b57be25accc229be19bef1e759ae"
-	;;
-"Ubuntu~focal")
-	hash="466f7307f16fc105a9ab00479d18bbcf3476666a6be2e28b7afb2a5143d0406c"
-	;;
+"bookworm")
```

Comment:
the correct naming is

`debian:bookworm`

you have to adapt to all this way.

refer to the sample here : https://github.com/pacstall/pacstall/wiki/Pacscript-101